File: JPAGiggity/src/main/java/com/skilldistillery/giggity/entities/BuyerReview.java
package com.skilldistillery.giggity.entities;

import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class BuyerReview {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;

	private String comment;

	private int rating;

	@Column(name = "job_id")
	private int jobId;

	@Column(name = "review_date")
	private LocalDateTime reviewDate;

}

File: ngGiggity/src/app/models/buyer-review.ts
export class BuyerReview {
  id: number;
  comment: string;
  rating: number;
  reviewDate: Date;
  constructor(
    id: number,
    rating: number,
    reviewDate: Date,
    comment?: string
  ) {
    this.id = id;
    this.comment = comment;
    this.rating = rating;
    this.reviewDate = reviewDate;
  }
}
